doc/requirements/occupancy_per_ioclass

```diff
@@ -0,0 +1,84 @@
+---
+group: Occupancy per ioclass
+---
+
+Requirements for occupancy per ioclass
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting occupancy limit
+id: occupancy_limit
+---
+
+It shall be possible to set a maximum size of an ioclass.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed occupancy values
+id: allowed_values
+---
+
+OpenCAS shall accept values for a max size of an ioclass in range from 0 to 100
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Occupancy limit interpreatation
+id: limit_interpretation
+---
+
+Max occupancy value shall be interpreted as a percentage of a cache size
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Default occupancy limit
+id: default_limit
+---
+
+On starting a cache anew the default's ioclass max occupancy shall be set to 100
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting an occupancy limit on a cache load
+id: limit_on_load
+---
+
+When a cache state is loaded, for each ioclass the occupancy limit shall be set
+to the value used before the cache has been shut down.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Display an occupancy limit value
+id: display_limit
+---
+
+For each configured ioclass it shall be possible to display a currently set
+value of the occupancy limit
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Respecting occupancy limit on a cache miss
+id: respecting_limit_on_miss
+---
+
+If an IO is a miss and a target ioclass has reached it's occupancy limt, before
+inserting a new data into a cache, equivalent number of cache blocks shall be
+evicted in the first place
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: IO to a disabled ioclass
+id: disabled_ioclass
+---
+
+IO shall be submitted in pass-through mode when a target's ioclass max occupancy
```

Comment:
1. IOs
2. the target ioclass's
Maybe change the order: When the target ioclass's max occupancy is set to 0, IOs shall be submitted in pass-through mode.